package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "app",
-  "version": "1.0.0",
-  "description": "",
+  "name": "mina-block-explorer",
```

Comment:
This npm repo is entirely for playwright. It's dev only. So perhaps name it `mina-block-explorer-tests` and make it private?